spec/specification/consumer_spec.rb

```diff
@@ -463,25 +463,37 @@ module Pod
 
       it 'returns the on demand resources specified as a string wrapped in an array' do
         @spec.on_demand_resources = { 'Maps' => 'MapView/Map/Resources/*.png' }
-        @consumer.on_demand_resources.should == { 'Maps' => ['MapView/Map/Resources/*.png'] }
+        @consumer.on_demand_resources.should == { 'Maps' => { :paths => ['MapView/Map/Resources/*.png'], :category => :on_demand } }
```

Comment:
consumer now normalizes all values to be a hash with 2 keys, :paths and :category in which :paths is always an array and :category a symbol.